### ISSUE TITLE
Problem with gray-scaled tiff file

### ISSUE THREAD

**Issue by muxmax** [haraldk/TwelveMonkeys#162]
Hello,

For the attached gay scaled tiff file an error occurs while loading:
javax.imageio.IIOException: Unsupported SamplesPerPixel/BitsPerSample combination for Bi-level/Gray TIFF (expected 1/1, 1/2, 1/4, 1/8 or 1/16): 2/8

When I debug to the point where the exception is thrown, I'm at TiffImageReader:line 353

Please rename the png-file to tiff.
![greyscaled](https://cloud.githubusercontent.com/assets/1533880/9384902/aaff5e5c-4753-11e5-9ba1-281d172e9e9d.png)

**Comment by haraldk** [comment]
Hi muxmax,

Thanks for reporting!

You have run into a TIFF containing grayscale + alpha channel. I've never seen one before, hence no support. Anyway, I want to support it in the future. I've already fixed the reader locally, I just need to clean it up a little before I push it. Hopefully I'll do it later today.

PS: I think I've also fixed the Maven build settings in the latest version, to enable line numbers in the stack traces. :-)


Best regards,

-- 
Harald K
